test/common/json-model-parser.spec.js

```diff
@@ -0,0 +1,189 @@
+describe('json-model-parser', function () {
+    'use strict';
+    let requireJs = require('requirejs');
+    requireJs.config({
+        baseUrl: __dirname
+    });
+    const ModelParser = requireJs('../../src/common/json-model-parser');
+    const fs = require('fs');
+    const path = require('path');
+    const assert = require('assert');
+
+    describe('basic-sequential', function () {
+        let sequentialModelConfig = null;
+        let layerConfigProperties = null;
+        before(() => {
+            layerConfigProperties = [
+                'bias_regularizer', 'dtype', 'bias_constraint',
+                'kernel_regularizer', 'name', 'trainable', 'bias_initializer',
+                'activity_regularizer', 'kernel_constraint',
+            ];
+        });
+
+        beforeEach(() => {
+            let sequentialModelTxt = fs.readFileSync(path.resolve(__dirname,
+                '../test-cases/modelJsons/sequential_dense.json'));
+            sequentialModelConfig = JSON.parse(sequentialModelTxt);
+        });
+
+        it('Should find the number of models to be 1', () => {
+            let numModels = ModelParser.countNumberOfModels(sequentialModelConfig);
+            assert.equal(numModels, 1);
+        });
+
+        it('Should flatten the nested model, although none of them exist', () => {
```

Comment:
What does "although none of them exist" mean?

Reply:
This is somewhat vague, I changed it to 
```
It should parse the sequential model, with no nested models
```